### PR TITLE
Added missing destroy method

### PR DESCRIPTION
**Pull request by myoung34**

### CODE REVIEW

File: app/controllers/adhoq/queries_controller.rb [hunk 1]
```diff
@@ -29,6 +29,12 @@ def update
       redirect_to @query
     end
 
+    def destroy
+      Adhoq::Query.find(params[:id]).destroy!
+      @queries = Adhoq::Query.recent_first
+      render 'adhoq/queries/index'
```
Comment (takkanm) [inline app/controllers/adhoq/queries_controller.rb]: Why don't you use redirect ?
Better to use the redirect, I think that it is good.

File: lib/adhoq/version.rb [hunk 2]
```diff
@@ -1,3 +1,3 @@
 module Adhoq
-  VERSION = "0.0.6"
+  VERSION = "0.1.0"
```
Comment (takkanm) [inline lib/adhoq/version.rb]: Please don't change the version.

**Comment by takkanm** [comment]
Thank you for your contribution. I commented two points. Please check it.

**Comment by myoung34** [comment]
@takkanm changed both.

**Comment by takkanm** [comment]
Thanks :+1: 
I will merged after I confirm to work in my local. :smile:

**Pull request merged**